scss/previews/_icon.scss

```diff
@@ -2,35 +2,47 @@
 // ============
 
 [data-sassdoc='icon-preview'] {
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: space-between;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
   margin: size('gutter') negative('shim');
+
+  @include above('page') {
+    grid-template-columns: repeat(auto-fill, minmax(30%, 1fr));
+  }
 }
 
 [data-sassdoc-icon] {
   color: color('text-light');
-  flex: 1 1 15%;
   margin: size('gutter') size('shim');
   min-width: size('rhythm') * 5;
 
   [data-icon] {
     @include square('h1');
+    margin-bottom: size('gutter');
   }
 }
 
 [data-sassdoc='icon-info'] {
   font-size: size('small');
-  padding: size('half-shim') 0;
+  position: relative;
   white-space: nowrap;
 }
 
 [data-sassdoc='icon-name'] {
   color: color('text');
   font-weight: bold;
-  text-transform: uppercase;
+  white-space: nowrap;
+}
+
+.herman-file-extension {
+  font-size: size('xsmall');
+  font-weight: normal;
 }
 
 [data-sassdoc='icon-path'] {
   @include font-family('code');
+  font-size: size('xxsmall');
+  text-align: left;
```

Comment:
Not sure this is needed. Maybe another tooltip issue?

Reply:
yep